desktop/FSharpWolfenstein.Desktop.fsproj

```diff
@@ -306,5 +307,6 @@
       <PackageReference Include="Silk.NET.Windowing" Version="2.15.0" />
       <PackageReference Include="SixLabors.ImageSharp" Version="1.0.4" />
       <PackageReference Include="SixLabors.ImageSharp.Drawing" Version="1.0.0-beta13" />
+      <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="6.0.0" />
```

Comment:
This line should be not needed once https://github.com/dotnet/Silk.NET/pull/959 lands in Silk.Net and they release new version.